### DISCUSSION TITLE
Working examples of snaps-sdk 6.0.0 components usage

### DISCUSSION THREAD

**Discussion by urosmeh** [MetaMask/snaps#2597] [Q&A]
I've been trying to find an example of a snap that uses jsx components but haven't found one what would work. snappy-airlines dont work for me. I also tried creating new snap project and installed needed packages and it also didn't work. @metamask/create-snap package installs old version that doesn't support jsx as well but installed version (4.0.0) components are deprecated. Can someone help me please?

**Comment by Montoya** [comment]
You need a pre-release build of Extension or Flask version 12. You can pick from this list: https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/92767/workflows/7e4564c6-684b-4d89-8ede-5d910f0aa022/jobs/3456184/artifacts

I recommend [metamask-flask-chrome-12.0.0-flask.0.zip](https://output.circle-artifacts.com/output/job/b88b4851-2ad5-4df5-95e1-3dddae9c000c/artifacts/0/builds-flask/metamask-flask-chrome-12.0.0-flask.0.zip)
